### PR TITLE
connectWifi Function

### PR DESCRIPTION
**Pull request by Suchismita99**
Code has been added to this function that will allow the MKR 1010 arduino to connect with the wifi

### CODE REVIEW

File: MKR_WIFI_1010/MKR_WIFI_1010_main/MKR_HeaderFile.h [hunk 1]
```diff
@@ -3,8 +3,31 @@
 File used to define the functions that will be needed
 as the project progesses.
  */
- 
+
+#include <WiFiNINA.h>
```
Comment (mohtashim-khan) [inline MKR_WIFI_1010/MKR_WIFI_1010_main/MKR_HeaderFile.h:7]: If this is included in the header file, we dont need it in the main file

File: MKR_WIFI_1010/MKR_WIFI_1010_main/MKR_HeaderFile.h [hunk 2]
```diff
@@ -3,8 +3,31 @@
...
+
  void initBluetooth();
  void intiWifi();
  void connectBluetooth();
- void connectWifi();
\ No newline at end of file
+void connectWifi(){
```
Comment (mohtashim-khan) [inline MKR_WIFI_1010/MKR_WIFI_1010_main/MKR_HeaderFile.h:12]: Header files are only for function declarations and other stuff like that, Lets put this function into a new file for now, like I have it for initializing the bluetooth, the code will be cleaner that way. Think of the header file like a blueprint

**Review by mohtashim-khan — CHANGES_REQUESTED** [review]
See comments

**Review by mohtashim-khan — COMMENTED** [review]
Also what are the DS_Store files used for? If not used for anything id get rid of them as well @Suchismita99

File: MKR_WIFI_1010/MKR_WIFI_1010_main/MKR_HeaderFile.h (hunk 2 above)
Comment (Suchismita99) [reply MKR_WIFI_1010/MKR_WIFI_1010_main/MKR_HeaderFile.h:12]: Okay, I understand. Did you already put my code in a new file or did you want me to do that?

Comment (Suchismita99) [reply MKR_WIFI_1010/MKR_WIFI_1010_main/MKR_HeaderFile.h:12]: Also, what is the difference between initwifi and connectwifi?

Comment (mohtashim-khan) [reply MKR_WIFI_1010/MKR_WIFI_1010_main/MKR_HeaderFile.h:12]: Ill change the file actually dont worry about it

**Pull request merged**